about.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/styles.css" />
+    <title>About Page</title>
+  </head>
+  <body>
+    <div class="container header_section">
+      <header class="header">
+        <nav id="navbar" class="header_toolbar">
+          <a href="./index.html" class="header_logo"
+            ><img src="./images/svg/MyLogo.svg" alt="My Logo"
+          /></a>
+          <div class="menu-wrap">
+            <button type="button" class="ham"></button>
+            <div>
+              <div class="navbar right-sidebar">
+                <ul>
+                  <li>
+                    <a href="./about.html" class="menu-link">About</a>
```

Comment:
- [ ] In your **about page** the navbar should contain a home link now you have an **about page link**.

![Capture web_17-11-2021_996_topeogunleye github io](https://user-images.githubusercontent.com/61562626/142161325-86b98c81-f380-4466-b0bd-7f7b1b078405.jpeg)